jest.config.js

```diff
@@ -2,6 +2,6 @@
 const { jest: jestConfig } = require('kcd-scripts/config')
 
 module.exports = Object.assign(jestConfig, {
-  roots: ['<rootDir>/src', '<rootDir>/test'],
-  testMatch: ['<rootDir>/test/**/*.(ts|tsx|js)']
+  roots: ['<rootDir>/src'],
+  testMatch: ['<rootDir>/**/__tests__/*.(ts|tsx|js)']
```

Comment:
I think this file can just be deleted now as these match the default jest config in `kcd-scripts`